service/src/tests.rs

```diff
@@ -182,9 +182,10 @@ mod tests {
     #[marine_test(config_path = "../Config.toml", modules_dir = "../artifacts/")]
     fn get_key_metadata_not_found() {
         clear_env();
-        let result = aqua_dht.get_key_metadata_cp("invalid_key".to_string(), 123u64, get_correct_timestamp_cp(1));
+        let key = "invalid_key".to_string();
+        let result = aqua_dht.get_key_metadata_cp(key.clone(), 123u64, get_correct_timestamp_cp(1));
         assert!(!result.success);
-        assert_eq!(result.error, "not found");
+        assert_eq!(result.error, f!("Requested key {key} does not exist on this peer"));
```

Comment:
@alari what do you think about it?

Reply:
It's expected to be a singleton service for each peer. If you can't find a key on this peer, then you go to another peer, not service.

From this PoV, to say that the key does not exist on this peer is better than speaking about a service, and the error message seems good.

The problem here is that we want to allow service reuse, e.g. to install AquaDHT service and to use it not to manage AquaDHT, but to manage some private lists of records, like participants list in the messenger room. In this case, speaking about a peer is strange.

Ideally, the message should say about the peer when the service is aliased, and about the service when it's not. Not sure how to do it.

Or it can speak about both, like "Requested key {key} does not exist in this service and/or peer".

Reply:
omg, why make it so complicated? 

`f!("Requested key {key} does not exist")`, that's it. The caller has the context, he knows what he calls, he has tetraplets and stuff.

No need to mess with scopes, entities, and fundamentals here. Like, if you try to read a file and it doesn't exist, it doesn't say "doesn't exist on this computer / FS / disk / universe / reality". It doesn't exist _as far as the program is concerned_

Reply:
And service _sure_ doesn't need to know if it's aliased or not...

Reply:
Well makes sense. I just want to ensure that users of the service will understand what exactly happens, as now I received claims that it's unclear.

Reply:
I hope we will have nice error handling patterns one day and will be able to throw errors and display them in the format of `{error} happened in service {tetraplet.service_id} at {tetraplet.peer_id}` on the JS/Rust Peer side